pages/plugins/using.md

```diff
@@ -47,8 +47,7 @@ location.
 
 ## Configuring plugins
 
-Plugins are configured using attributes in your pipeline YAML configuration. The simplest plugin is one that accepts no configuration, such as the [Library Example plugin](https://github.com/buildkite-plugins/library-example-buildkite-plugin):
-
+Plugins are configured using attributes on steps in your pipeline YAML definition. While you can't define plugins at a pipeline level, you can use [YAML anchors](/docs/plugins/using#using-yaml-anchors-with-plugins) to avoid repeating the plugin code over multiple steps. The simplest plugin is one that accepts no configuration, such as the [Library Example plugin](https://github.com/buildkite-plugins/library-example-buildkite-plugin):
```

Comment:
Need to add a blank line to fix how this displays:

<img width="858" alt="Screenshot 2023-04-04 at 4 47 54 pm" src="https://user-images.githubusercontent.com/119824349/229689571-32775787-bd29-40ce-9659-e4e0dbaa59a6.png">

```suggestion
Plugins are configured using attributes on steps in your pipeline YAML definition. While you can't define plugins at a pipeline level, you can use [YAML anchors](/docs/plugins/using#using-yaml-anchors-with-plugins) to avoid repeating the plugin code over multiple steps. The simplest plugin is one that accepts no configuration, such as the [Library Example plugin](https://github.com/buildkite-plugins/library-example-buildkite-plugin):

```